.github/ISSUE_TEMPLATE/good_first_issue.yml

```diff
@@ -0,0 +1,37 @@
+name: Good First Issue
+title: "<title>"
+description: Create a good first issue to help new contributors
+labels: ["good first issue"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the sections below to properly describe the new feature or the bug.
+
+        **Contribution Guidelines**
+
+        - If you want to work on an issue, please let us know in a comment below.
+        - Do not begin work on the issue unless it has been assigned to you.
+        - If you do not respond in 3 days, the issue may be reassigned to someone else.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the feature/bug
+      placeholder: A clear and concise description.
+  - type: textarea
+    id: rationale
+    attributes:
+      label: Steps to reproduce bug/implement feature
+      placeholder: Clear and concise steps to reproduce the bug or implement the feature.
+  - type: textarea
+    id: context
+    attributes:
+      label: Resources
+      placeholder: |
+        Add links or screenshots to help new contributors understand the issue.
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Additional Information
+      options:
+        - label: I'd like to work on this issue
```

Comment:
Remove this field as well :)

Reply:
From all the templates or only good first issue template?

Reply:
Only this template.